src/main/java/com/diamondfire/helpbot/bot/command/impl/stats/top/DiscordBoostersCommand.java

```diff
@@ -48,22 +49,34 @@ public void run(CommandEvent event) {
         Guild guild = event.getGuild();
         PresetBuilder preset = new PresetBuilder()
                 .withPreset(
-                        new InformativeReply(InformativeReplyType.INFO, "Current Discord Boosters", null)
+                        new InformativeReply(InformativeReplyType.INFO, "Discord Server Boosters", null)
                 );
         EmbedBuilder embed = preset.getEmbed();
         embed.setThumbnail("https://cdn.discordapp.com/emojis/699936318398136341.png?v=1");
         
         event.getGuild().findMembers((member -> member.getTimeBoosted() != null)).onSuccess((members) -> {
-            embed.setDescription(StringUtil.listView("> ", true, members.stream()
-                    .sorted(Comparator.comparing(Member::getTimeBoosted))
-                    .map(Member::getEffectiveName)
-                    .toArray(String[]::new)));
+            members.sort(Comparator.comparing(Member::getTimeBoosted));
+            embed.setDescription("A list of members that are currently boosting this Discord server.\n\n"+
+                    getFormattedBoosters(members));
             embed.setColor(new Color(255, 115, 250));
             event.reply(preset);
             guild.pruneMemberCache();
         });
     }
     
-}
-
-
+    private String getFormattedBoosters(List<Member> members) {
+        if (members.size() == 0) return "*None*";
+        else {
+            SimpleDateFormat format = new SimpleDateFormat("y'y'M'm'D'd'");
+            String result = "";
+            
+            for (Member member : members) {
+                String timeBoosted = format.format(member.getTimeBoosted());
+                result += "<@" + member.getId() + "> - " + timeBoosted + "\n";
+            }
+            
+            return result;
+        }
+    }
```

Comment:
This is a more of a readability opinion, but instead of having a result and appending newline do something like this.
```java
List<String> elements = new ArrayList<>();
for (Member member : members) {
    String timeBoosted = ....
    elements.add(member.getAsMention() + " - " + timeBoosted);
}

return Strings.join("\n", elements);
```



```

Reply:
And make this method private static